src/integration.test.ts

```diff
@@ -6,18 +6,17 @@ import { pki } from "node-forge";
 const { publicKey, privateKey } = pki.ed25519.generateKeyPair();
 const client = new SkynetClient("https://siasky.dev");
 
-const appID = "HelloWorld";
+const datakey = "HelloWorld";
 const json = { data: "thisistext" };
 
 // skip - used to verify end-to-end flow
-describe.skip("siasky.dev end to end", () => {
+describe("siasky.dev end to end", () => {
```

Comment:
skip that test please, it should be run only manually, otherwise there is a good chance it will fail on our CI and it's not a unit test